Update log-analytics-dashboards.md

ehissey opened the pull request:
Updating upgrade note at the top to be more clear that this is for Log Analytics My Dashboard, not Azure Dashboards.

azurecla commented:
Hi __@harelbr__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ehissey committed:
Update log-analytics-dashboards.md

Updating upgrade note at the top to be more clear that this is for Log Analytics My Dashboard, not Azure Dashboards.

GitHubber17 committed:
Update log-analytics-dashboards.md
